commands/chain_daemon_test.go

```diff
@@ -16,6 +16,23 @@ import (
 	"gx/ipfs/QmPVkJMTeRC6iBByPWdrRkD3BE5UXsj5HPzb4kPqL186mS/testify/require"
 )
 
+func TestChainHead(t *testing.T) {
+	t.Parallel()
+	require := require.New(t)
+	assert := assert.New(t)
+
+	d := th.NewDaemon(t).Start()
+	defer d.ShutdownSuccess()
+
+	op := d.RunSuccess("chain", "head")
+	result := op.ReadStdoutTrimNewlines()
+
+	maybeCid, err := cid.Decode(result)
+	require.NoError(err)
+
+	assert.Equal(result, maybeCid.String())
```

Comment:
this doesnt test that the output is right if there are multiple blocks in the tipset...

Reply:
I have filed an issue (https://github.com/filecoin-project/go-filecoin/issues/2398) to address the lack of test coverage, is testing this inside the scope of the PR?

Reply:
I don't know how to get TestDaemon into a state where it has a multiblock tipset, any ideas?

Reply:
it is not obvious or easy, skip for now